docs/i18n-legacy.md

```diff
@@ -0,0 +1,52 @@
+# How to provide `t` function to a class component
```

Comment:
I think we can remove the whole file?

Reply:
This is useful for class components. Unless/until move to hooks completely (including for lifecycle methods and state management).

Perhaps instead of _legacy_ it should be called _deprecated_ or _use only sometimes_.

Reply:
Oh gotcha, just having a notice like what you just wrote would be useful to make it clear 👍

Reply:
69c7f6978d07ea2a06750b5e43913de7d686dda1